luma/core/device.py

```diff
@@ -10,6 +11,8 @@
 import luma.core.const
 from luma.core.interface.serial import i2c, noop
 
+__all__ = ["fb"]
```

Comment:
doesn't this limit the imports to just `fb`; and thus does this not break importing other classes from this module?

Reply:
so perhaps move this new `fb` class to a separate (new) module?

Reply:
I had to recheck, but according to https://docs.python.org/3/tutorial/modules.html#importing-from-a-package:

> ... if a package’s `__init__.py` code defines a list named `__all__`, it is taken to be the list of module names that should be imported when `from package import *` is encountered ...

Hence it should not break imports because we never do wildcard imports in any sub-modules, and end-user code should never have to import either the device, parallel_device or dummy classes.

Reply:
thanks for looking into it, sounds good.